lib/cubits/login_state_cubit.dart

```diff
@@ -24,21 +27,21 @@ class LoginStateCubit extends Cubit<RequestState> {
 
```

Comment:
separate this inside if() into a variable in order to tell us better what this condition means